cmd/infra/powervs/validate.go

```diff
@@ -180,6 +180,8 @@ func validateCloudConnectionInPowerVSZone(name string, client *instance.IBMPIClo
 		return
 	}
 
+	// main objective here is to validate the number of cloud connection and return an error if that does not satisfy the condition
+	// so explicitly setting err to nil if cloud connection count condition is ok since err obj might hold error returned from previous func call
```

Comment:
can we clean the err here in line 182, then avoid the else?

Reply:
Done